.github/workflows/main.yml

```diff
@@ -70,3 +70,12 @@ jobs:
     - name: Run clib test
       if: matrix.rust_version == 'stable'
       run: sudo make check -C test/clib
+  
+  container:
+    name: container
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Test nispor-in-container
+      run: tools/nispor-in-container -- echo success
```

Comment:
This CI just invoked a container and do nothing but echo.

I guess `make check_in_container` should be used here.

Reply:
I knew there were a reason why I didn't do that way: inside the container, there are a few tests that fails because netdevsim cannot be configured from the container. So we cannot do `make check_in_container` in the CI. It is OK if I revert the last change and leave `tools/nispor-in-container -- echo success`? That will check that nispor-in-container script is OK.